MODULE.bazel

```diff
@@ -1,15 +1,10 @@
 module(
     name = "grpc-proto",
     version = "0.0.0",
+    bazel_compatibility = [">=7.0.0"],
```

Comment:
unless this is because of this repo, shouldn't this be coming from the transitives? just matters in case the transitives change that requirement